Formula/newsboat.rb

```diff
@@ -0,0 +1,29 @@
+class Newsboat < Formula
+  desc "RSS/Atom feed reader for text terminals"
+  homepage "https://newsboat.org/"
+  head "https://github.com/newsboat/newsboat.git"
+
+  stable do
```

Comment:
You can remove this `stable do` block and put the `url` and `sha256` up in the class scope -- it was only necessary because of the patch in the old formula.

Reply:
Done